### PR TITLE
Add Indonesia locale

### PR DESCRIPTION
**Pull request by emhaye**

### CODE REVIEW

**Comment by ordex** [comment]
Hi @emhaye and thanks for your contribution!
Please, don't forget to also modify the LINGUAS file as has been done in [this other PR](https://github.com/axel-download-accelerator/axel/pull/115/files)

**Comment by emhaye** [comment]
@ordex  Thank's i will modified it

File: po/id.po [hunk 1]
```diff
@@ -12,7 +12,7 @@ msgstr ""
 "PO-Revision-Date: 2017-10-02 14:06+0700\n"
 "Last-Translator: Mahyuddin <mhy> <yudi.al@gmail.com>\n"
 "Language-Team: Indonesian (Indonesia) <yudi.al@gmail.com>\n"
-"Language: id_ID\n"
+"Language: id\n"
```
Comment (ordex) [inline po/id.po]: Please, don't fix non-merged commits with a new patch. Rather fix the wrong commit.
You can do this using `git rebase`

Cheers,

Comment (ordex) [reply po/id.po]: Note that this change has to go in the LINGUAS file too.

Comment (emhaye) [reply po/id.po]: Oke, i will use new pull-request.

Comment (ordex) [reply po/id.po]: You can just push in the same branch (with --force) and the current pull request will get updated. No need to create a new one.

Comment (emhaye) [reply po/id.po]: Thank's

**Comment by ordex** [comment]
@eribertomota can you please have a look at this PR when you have time? Thanks!

File: po/id.po [hunk 2]
```diff
@@ -0,0 +1,475 @@
+# Italian translations for axel package.
```
Comment (ismaell) [inline po/id.po]: Please fix the description and copyright.

File: po/id.po [hunk 3]
```diff
@@ -0,0 +1,475 @@
...
+
+#: src/axel.c:470
+#, c-format
+msgid "Error while waiting for connection: %s"
+msgstr "Galat saat menunggu koneksi: %s"
+
+#: src/axel.c:502
+#, c-format
+msgid "Connection %i timed out"
+msgstr "Sambungan %ihabis waktunya"
```
Comment (ismaell) [inline po/id.po]: Is this string right? Seems to be missing a space after `%i`.

Comment (emhaye) [reply po/id.po]: I see, i will fix

**Comment by ismaell** [comment]
Merged.

**Pull request closed without merging**